src/views/testUtils.js

```diff
@@ -17,7 +17,7 @@ export const visit = path => {
     // From the docs: Note that, even though goto normally waits
     // for all the resources on a page to load, a timeout exception
     // is only raised if the DOM itself has not yet loaded.
-    gotoTimeout: 4000
+    gotoTimeout: process.env.TEST_TIMEOUT || 4000
```

Comment:
@kebien6020 I actually am having the issue now on `ServiceRequest.test.js` while trying to commit the changes you suggested. I can bump the jasmine timeout, and I can't pass the test. I just tried it with 40000 and it still fails with the error from #32.